### PR TITLE
Rename "supercluster" to "clusterset"

### PR DESCRIPTION
**Pull request by skitt**
The latter is the name chosen by the multi-cluster SIG.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by nyechiel** [comment]
(https://github.com/submariner-io/lighthouse/pull/229 is the Lighthouse PR)

**Comment by nyechiel** [comment]
FWIW, I think we should not merge this PR before there is a final Ack from the multi-cluster SIG that this is indeed the chosen term.

**Comment by skitt** [comment]
> FWIW, I think we should not merge this PR before there is a final Ack from the multi-cluster SIG that this is indeed the chosen term.

The official choice was just made in the SIG call, it’s “clusterset”.

**Comment by dfarrell07** [comment]
> > FWIW, I think we should not merge this PR before there is a final Ack from the multi-cluster SIG that this is indeed the chosen term.
> 
> The official choice was just made in the SIG call, it’s “clusterset”.

Official announcement: https://groups.google.com/forum/#!msg/kubernetes-sig-multicluster/5QoyXmEK7oU/YLZPZJ2ZBAAJ

File: src/content/troubleshooting/_index.en.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Spec:
   Ports:                    <nil>
   Session Affinity:
   Session Affinity Config:  <nil>
-  Type:                     SuperclusterIP
+  Type:                     ClusterSetIP
```
Comment (skitt) [inline src/content/troubleshooting/_index.en.md:156]: This is pending confirmation from the SIG wrt. the MCS API.

Comment (skitt) [reply src/content/troubleshooting/_index.en.md:156]: The change has been confirmed, see https://github.com/JeremyOT/mcs-api/commit/b44bc782c2c1af504d4e43b4f844d7ce004db231

**Review by aswinsuryan — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]
This is ready, but let's wait until we are very close to release so we don't confuse existing users testing it.

**Review by vthapar — APPROVED** [review]

**Pull request merged**